examples/doxygen/block_dynamic_sparsity_pattern.cc

```diff
@@ -54,7 +54,7 @@ int main()
   constraints.close();
 
   std::vector<unsigned int> dofs_per_block(fe.n_blocks());
-  DoFTools::count_dofs_per_block(dof, dofs_per_block);
+  DoFTools::count_dofs_per_fe_block(dof, dofs_per_block);
```

Comment:
this looks incorrect.

Reply:
This bug would have been caught by [[nodiscard]], right? :-)

Reply:
Does `clang-tidy` check files in `examples/doxygen`?

Reply:
No, that's the problem -- this piece of code wouldn't have compiled at all. I replaced all occurrences of the function name and then compiled the library and all examples and tests, but this file is simply not compiled.

Reply:
Fix this.